README.rst

```diff
@@ -537,6 +537,10 @@ Choose from the list of available fixers:
                         with at least one whitespace
                         around.
 
+* **echo_to_print** [contrib]
+                        Converts echo functions to
```

Comment:
`print is not actually a real function (it is a language construct) `
so yeah ;)